README.md

```diff
@@ -1,17 +1,32 @@
-# mcrouter
-[![Build Status](https://travis-ci.org/evertrue/mcrouter-cookbook.svg)](https://travis-ci.org/evertrue/mcrouter-cookbook)
```

Comment:
Is it that it's broken due to code, or due to pipeline setup? 
Last run (https://travis-ci.org/evertrue/mcrouter-cookbook/jobs/265735410) had some failure logs with it.